### PR TITLE
fix: replace imgur with s3 for Portuguese guide without conflict

### PR DESCRIPTION
**Pull request by ahmaxed**
(cherry picked from commit 55e786f8dbcf917e16088be97e18c951d937cd71)

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Reference #36007

### CODE REVIEW

File: guide/portuguese/cplusplus/index.md [hunk 1]
```diff
@@ -78,8 +79,8 @@ int main()
 *   Não se esqueça de informar ao computador que este é o fim da função main (). Para fazer isso, você adiciona a chave de fechamento "}". Você encontrará um erro antes da execução do programa se você não incluir o **}** .
 
 ### O código deve ser algo como isto:
-
-![Img](https://i.imgur.com/d1liGwI.png)
+https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/portuguese/cplusplus/index.md]: Is this line needed?
```suggestion
```

File: guide/portuguese/cplusplus/tokens-variables/index.md [hunk 2]
```diff
@@ -21,8 +22,8 @@ Tokens são as menores unidades de um programa que são importantes para o compi
 ### O que são variáveis?
 
 *   Definição de livro de texto: As variáveis ​​são nomeadas locais de memória cujos dados podem ser alterados.
-    
-*   Mas eu gostaria que você pensasse em uma variável para ser algo como uma caixa, algo assim: ![Img](https://i.imgur.com/YdbgWHL.png)
+    https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/portuguese/cplusplus/tokens-variables/index.md]: Is this line needed?
```suggestion
```

File: guide/portuguese/cplusplus/tokens-variables/index.md [hunk 3]
```diff
@@ -176,8 +177,8 @@ Exemplo:
 Se uma variável não for inicializada, ela contém um valor de lixo. Por exemplo:
 
 Então, em termos de caixas, você pode imaginar isso como -
-
-![Img](https://i.imgur.com/YdbgWHL.png)
+https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/portuguese/cplusplus/tokens-variables/index.md]: Is this line needed?
```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@ahmadabdolsaheb See my suggested changes.

**Comment by ahmadabdolsaheb** [comment]
@RandellDawson, this should be ready now.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb OK, this PR looks good now.

**Comment by ahmadabdolsaheb** [comment]
i made a few changes, so This should be ready to go.

**Comment by ahmadabdolsaheb** [comment]
@thecodingaviator, I would appreciate it If you could take a look into this.

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**